services/ui-src/src/components/fields/DropdownField.tsx

```diff
@@ -177,6 +179,7 @@ export const DropdownField = ({
         name={name}
         id={name}
         label={labelText || ""}
+        ariaLabel={labelText ? undefined : name}
```

Comment:
Fixes `Please provide an 'ariaLabel' when using the <Dropdown> component without a 'label' prop.`